packages/amplify-codegen-e2e-tests/test-apps/ts/scripts/set-schema.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
```

Comment:
Is this script in use? I see there is a method `updateApiSchemaWithText` in e2e test used for setting up the schema.

Reply:
This is used for local testing outside of the test suite. It is not strictly necessary, but I found it useful.